.rubocop.yml

```diff
@@ -122,6 +122,10 @@ Style/TrailingBlankLines:
 Style/TrailingWhitespace:
   Enabled: true
 
+# Checks for uses of double negation (!!) to convert something to a boolean value. As this is both cryptic and usually redundant, it should be avoided.
```

Comment:
I think this can be removed: _As this is both cryptic and usually redundant, it should be avoided._

It is already clear with the first sentence and it makes it too long. If someone wants more information can check Rubocop documentation.